aixplain/modules/benchmark.py

```diff
@@ -82,17 +82,17 @@ def check_status(self):
         self._update_from_response(response)
         return self.status
     
-    def download_results_as_csv(self, save_path: Optional[Text] = None, returnDataFrame: Optional[bool] = False):
+    def download_results_as_csv(self, save_path: Optional[Text] = None, return_dataframe: Optional[bool] = False):
```

Comment:
when the default of a parameter is not `None`, there is no need to use Optional:

```
return_dataframe: bool = False
```